industrial_robot_client/package.xml

```diff
@@ -34,4 +34,5 @@
   <run_depend>urdf</run_depend>
   <run_depend>industrial_msgs</run_depend>
   <run_depend>industrial_utils</run_depend>
+  <test_depend>roslaunch</test_depend>
```

Comment:
I seem to remember that for package manifest v1, this should actually be a `build_depend`. See https://github.com/ros-industrial/fanuc/issues/76 fi.